src/components/generic/LogoBtn.tsx

```diff
@@ -0,0 +1,33 @@
+interface BtnProps {
+    name: string;
+    onClick: (event: React.MouseEvent<HTMLButtonElement>) => void;
+}
+
+const LogoBtn = ({ name, onClick }: BtnProps) => {
+    return (
```

Comment:
this would be a bit simpler if you used a simple object to map name to image. For example:
```
{
  play: 'play-svgrepo-com.svg',
  pause: 'pause-alt-svgrepo-com.svg'
 ...
}
```